From 2dc0dec7ea74200f26a0db254d4a1e641f29c5e5 Mon Sep 17 00:00:00 2001
From: Felix Kuehling <Felix.Kuehling@amd.com>
Date: Mon, 31 Jul 2017 17:16:01 -0400
Subject: [PATCH 5415/5855] drm/amdkfd: Rename topology_device_by_nodeid

To kfd_topology_device_by_proximity_domain. That's a better
description of what it does, and the naming convention matches
similar kfd_topology functions.

Change-Id: I33043eadefe6500f2218d0cae61eba7cf96e4f53
Signed-off-by: Felix Kuehling <Felix.Kuehling@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_crat.c     | 2 +-
 drivers/gpu/drm/amd/amdkfd/kfd_priv.h     | 3 ++-
 drivers/gpu/drm/amd/amdkfd/kfd_topology.c | 5 +++--
 3 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_crat.c b/drivers/gpu/drm/amd/amdkfd/kfd_crat.c
index d44fd2c..ac7eaac 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_crat.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_crat.c
@@ -340,7 +340,7 @@ static int kfd_parse_subtype_iolink(struct crat_subtype_iolink *iolink,
 	 * At this time, also add the corresponded CPU->GPU link.
 	 */
 	if (props && props->iolink_type == CRAT_IOLINK_TYPE_PCIEXPRESS) {
-		cpu_dev = topology_device_by_nodeid(id_to);
+		cpu_dev = kfd_topology_device_by_proximity_domain(id_to);
 		if (!cpu_dev)
 			return -ENODEV;
 		/* same everything but the other direction */
diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_priv.h b/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
index 9049764..cb51c0a 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
@@ -846,7 +846,8 @@ int kfd_topology_init(void);
 void kfd_topology_shutdown(void);
 int kfd_topology_add_device(struct kfd_dev *gpu);
 int kfd_topology_remove_device(struct kfd_dev *gpu);
-struct kfd_topology_device *topology_device_by_nodeid(uint32_t node_id);
+struct kfd_topology_device *kfd_topology_device_by_proximity_domain(
+						uint32_t proximity_domain);
 struct kfd_dev *kfd_device_by_id(uint32_t gpu_id);
 struct kfd_dev *kfd_device_by_pci_dev(const struct pci_dev *pdev);
 struct kfd_dev *kfd_device_by_kgd(const struct kgd_dev *kgd);
diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_topology.c b/drivers/gpu/drm/amd/amdkfd/kfd_topology.c
index b544ca5..89876bb 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_topology.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_topology.c
@@ -43,7 +43,8 @@ static struct kfd_system_properties sys_props;
 static DECLARE_RWSEM(topology_lock);
 static atomic_t topology_crat_proximity_domain;
 
-struct kfd_topology_device *topology_device_by_nodeid(uint32_t node_id)
+struct kfd_topology_device *kfd_topology_device_by_proximity_domain(
+						uint32_t proximity_domain)
 {
 	struct kfd_topology_device *top_dev;
 	struct kfd_topology_device *device = NULL;
@@ -51,7 +52,7 @@ struct kfd_topology_device *topology_device_by_nodeid(uint32_t node_id)
 	down_read(&topology_lock);
 
 	list_for_each_entry(top_dev, &topology_device_list, list)
-		if (top_dev->proximity_domain == node_id) {
+		if (top_dev->proximity_domain == proximity_domain) {
 			device = top_dev;
 			break;
 		}
-- 
2.7.4

